### ISSUE TITLE
Deployment error

### ISSUE THREAD

**Issue by diszmaeu** [aws-quickstart/quickstart-salesforce-connect-appsync-rds-postgresql#23]
Hello,

I'm getting an error when deploying the resource RDSResolverLambda

Resource handler returned message: "Error occurred while GetObject. S3 Error Code: NoSuchKey. S3 Error Message: The specified key does not exist. (Service: Lambda, Status Code: 400)" 

I'm filling in all parameters and leaving the default ones untouched. I would appreciate any guidance.

Thanks,

**Comment by kishd** [comment]
Hello,
It's likely an issue with your GraphQLSchemaLocation or RDSSchemaLocation parameter. Can you share the values for these 2 parameters?

**Comment by diszmaeu** [comment]
Hi kishd, thanks for your help. I was actually using the values suggested on [this](https://github.com/aws-quickstart/quickstart-salesforce-connect-appsync-rds-postgresql/issues/20) other issue. Here are the values:

GraphQL schema location - https://raw.githubusercontent.com/aws-quickstart/quickstart-salesforce-connect-appsync-rds-postgresql/main/samples/postgres-schema.graphql
GraphQL schema name - postgres-schema.graphql
RDS schema location - https://raw.githubusercontent.com/aws-quickstart/quickstart-salesforce-connect-appsync-rds-postgresql/main/samples/postgres-ddl.sql
RDS schema name - postgres-ddl.sql

I'm keeping default values for Partner Solution S3 bucket name, Partner Solution S3 key prefix and Partner Solution S3 bucket Region.

I tried again and I see the same error message.

I tried once again with the above values and
